src/math_physics/time_system/test_date_time_format.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "date_time_format.hpp"
 
+using namespace time_system;
```

Comment:
[IMO] 明示的に
```suggestion
namespace time_system {
```
と `time_system` 名前空間内に実装を置いたほうが適切だと思います。